rs/bitcoin/ckbtc/minter/src/storage/tests.rs

```diff
@@ -0,0 +1,48 @@
+use crate::state::eventlog::{Event, EventType};
+use crate::test_fixtures::{ignored_utxo, ledger_account};
+
+#[test]
+fn should_decode_encoded_event() {
```

Comment:
Maybe it would be a good exercise for you to transform those tests into a [proptest](https://altsysrq.github.io/proptest-book/) to get coverage on all event types. Most of the work would be to generate an arbitrary `EventType`, similar to what was done [here](https://github.com/dfinity/ic/blob/49529d2758b87f5a32ce744972f72d9d3014ee68/rs/ethereum/cketh/minter/src/state/tests.rs#L738) and could be done in a separate PR. WDYT?